src/main/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -137,6 +137,11 @@ public static void main(final String[] argv) {
     @Override
     protected void setup(final SAMFileHeader header, final File samFile) {
 
+        // No ribosomal intervals and rRNA fragment percentage = 0
+        if ( (RIBOSOMAL_INTERVALS == null || RIBOSOMAL_INTERVALS.length() == 0) && RRNA_FRAGMENT_PERCENTAGE == 0 ) {
+            throw new PicardException("Must use a RIBOSOMAL_INTERVALS file if RRNA_FRAGMENT_PERCENTAGE = 0.0");
+        }
+
```

Comment:
- Moved `No ribosomal intervals and rRNA fragment percentage = 0`  Overrode `customCommandLineValidation`
- Removed the empty file check, not needed. The code throws an IllegalStateException if the header is missing.
- If the ribosomal intervals file is missing intervals, an exception is thrown from `makeOverlapDetector`. Just having a waning with `RRNA_FRAGMENT_PERCENTAGE = 0.0` have [this if statement](https://github.com/broadinstitute/picard/blob/master/src/main/java/picard/analysis/directed/RnaSeqMetricsCollector.java#L164) always true